### DISCUSSION TITLE
/mmengine/visualization/utils.py:245: UserWarning: Glyph  20998 (\N{CJK UNIFIED IDEOGRAPH-5206}) missing from current font.

### DISCUSSION THREAD

**Discussion by htaoruan** [open-mmlab/mmocr#1797] [General]
In the last version of MMOCR, I execute the inference code and get the following output：
![image](https://user-images.githubusercontent.com/113572928/226789964-cbe3fc1f-c65b-4ecf-8f0a-90cde9467869.png)

**Comment by gaotongxiao** [comment]
See this: https://github.com/open-mmlab/mmocr/pull/1709

**Reply by htaoruan** [reply]
Thanks
